assets/semi-structured/clickhouse/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:latest
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    curl \
+    gpg
+
+## Import ClickHouse key
+RUN curl -fsSL 'https://packages.clickhouse.com/rpm/lts/repodata/repomd.xml.key' \
+    | gpg --dearmor -o /usr/share/keyrings/clickhouse-keyring.gpg \
+    && echo "deb [signed-by=/usr/share/keyrings/clickhouse-keyring.gpg]" \
+    "https://packages.clickhouse.com/deb stable main" \ 
+    | tee /etc/apt/sources.list.d/clickhouse.list
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    python3-venv \
+    tmux \
+    vim \
+    libcurl4 \
+    libcurl4-openssl-dev \
+    clickhouse-server=23.3.1.2823 \
+    clickhouse-client=23.3.1.2823 \
+    clickhouse-common-static=23.3.1.2823 \
```

Comment:
Not suggesting we change, but what's the difference between installing ClickHouse in a container and setting the base image (`FROM xxx`) to the existing [ClickHouse container](https://hub.docker.com/r/clickhouse/clickhouse-server/)?

Reply:
Good point. Because I didn't realize that there is an clickhouse container lol. Should be fixed in the future.